File: R/Introduccion/Calculadora.R
> imprimir_menu <- function(){
  +     cat("Menu de opciones:\n")
  +     cat("1. Suma\n")
  +     cat("2. Resta\n")
  +     cat("3. Multiplicación\n")
  +     cat("4. División\n")
  +     cat("5. Salir\n")
  + }
> 
  > calculadora_R <- function(){
    +     while (TRUE) {
      +         imprimir_menu()
      +         opcion <- as.integer(readline(prompt="Selecciona una opción: "))
      +         if(opcion==5){
        +             cat("Saliendo de la calculadora...\n")
        +             break
        +         }
      +         x <- as.numeric(readline(prompt="Ingresa el primer número: "))
      +         y <- as.numeric(readline(prompt="Ingresa el segundo número: "))
      +         if(opcion==1){
        +             cat("El resultado es: ", x+y,"\n")
        +         }else if(opcion==2){
          +             cat("El resultado es: ", x-y,"\n")
          +         }else if(opcion==3){
            +             cat("El resultado es: ", x*y,"\n")
            +         }else if(opcion==4){
              +             if (y != 0) {
                +                 cat("El resultado es: ", x/y,"\n")
                +             } else {
                  +                 cat("No se puede dividir por cero.\n")
                  +             }
              +         }else{
                +             cat("Opción inválida. Intenta de nuevo.\n")
                +         }
      +     }
    + }
  > 
    > calculadora_R()
  

File: Python/Introduccion/Calculadora.py
def suma(x, y):
    return x + y

def resta(x, y):
    return x - y

def multiplicacion(x, y):
    return x * y

def division(x, y):
    if y == 0:
        return "Error: No se puede dividir por cero"
    return x / y

def calculadora():
    print("Selecciona una operación:")
    print("1. Suma")
    print("2. Resta")
    print("3. Multiplicación")
    print("4. División")

    while True:
        opcion = input("Ingresa el número de la operación que deseas realizar (1/2/3/4): ")

        if opcion in ('1', '2', '3', '4'):
            num1 = float(input("Ingresa el primer número: "))
            num2 = float(input("Ingresa el segundo número: "))

            if opcion == '1':
                print("Resultado:", suma(num1, num2))
            elif opcion == '2':
                print("Resultado:", resta(num1, num2))
            elif opcion == '3':
                print("Resultado:", multiplicacion(num1, num2))
            elif opcion == '4':
                print("Resultado:", division(num1, num2))

            break
        else:
            print("Opción no válida. Por favor, ingresa un número válido (1/2/3/4).")

calculadora()